id-authentication-default.properties

```diff
@@ -665,4 +666,14 @@ mosip.ida.authentication.error.eventing.encrypt.partner.id=mpartner-default-temp
 mosip.ida.kafka.bootstrap.servers=kafka-0.kafka-headless.${kafka.profile}:${kafka.port},kafka-1.kafka-headless.${kafka.profile}:${kafka.port},kafka-2.kafka-headless.${kafka.profile}:${kafka.port}
 spring.kafka.admin.properties.allow.auto.create.topics=true
 logging.level.org.apache.kafka=DEBUG
+
+#Spring boot 3.x onwards we need to specify the below property to unmask values in actuator env url
+management.endpoint.env.show-values=ALWAYS
+spring.mvc.trailing-slash-match=false
+spring.batch.jdbc.initialize-schema=embedded
+spring.codec.max-in-memory-size=500KB
+spring.servlet.multipart.max-file-size=10MB
+spring.servlet.multipart.max-request-size=10MB
```

Comment:
this property is required, if required they were we are using multipart file in ida

Reply:
https://stackoverflow.com/a/47947964/13425606
spring.batch.jdbc.initialize-schema=embedded
As of spring 2.5.0, we should use spring.batch.jdbc.initialize-schema

Reply:
spring.codec.max-in-memory-size=500KB
spring.servlet.multipart.max-file-size=10MB
spring.servlet.multipart.max-request-size=10MB

Removed these properties.